### ISSUE TITLE
PDF build fails or not, depending on the directory name it's in

### ISSUE THREAD

**Issue by oddhack** [asciidoctor/asciidoctor-pdf#1888]
TL;DR we have a pretty complex document build which I'm integrating into Github Actions. Actions extracts the repository under /__w/SYCL-Docs/SYCL-Docs/ - I don't think I can control this location - and on this path, I see lots of errors like this:

> asciidoctor: WARNING: image to embed not found or not readable: /\<em>w/SYCL-Docs/SYCL-Docs/adoc/out/equations_temp/stem-5373936b5901139f132b498255a95fa5.svg

(see here: https://github.com/KhronosGroup/SYCL-Docs/pull/117/checks?check_run_id=2066711877)

This is not Docker-specific, but path-specific: running the build on a repo located under /__w/ results in the errors and putting it under a more common path like /sycl does not get errors.

The '/\<em>w/' part of the error suggests that at some point in the toolchain, the '__w' may be getting interpreted as asciidoctor markup rather than as a literal pathname. I have tried replicating the error using just a regular image: directive but have not been able to, which leads me to think something about the way asciidoctor-pdf is interacting with asciidoctor-mathematical is responsible, rather than general handling of paths in image: directives.

If you need to reproduce our build to diagnose the problem, here's a recipe assuming you can run docker:

```
# In your shell:
docker run -it --rm khronos-group/docker-images:asciidoctor-spec /bin/bash

# Inside the Docker image, build under /__w/ - should fail
mkdir -p /__w/SYCL-Docs
cd /__w/SYCL-Docs
git clone https://github.com/KhronosGroup/SYCL-Docs.git
cd SYCL-Docs/adoc
git checkout 9cc7b456ab2ef288a3f758634dc24a4d0e715fee
make QUIET= pdf
# Lots of 'image to embed not found or readable errors' followed by build failing

# Now build under /sycl/ - should work
make clean
mv /__w /sycl
cd /sycl/SYCL-Docs/SYCL-Docs/adoc
make pdf
# No errors and build succeeds
```

If this is too complex I can try to strip down the document markup - the problem appears to happen at the first latexmath:[] directive in the doc.
